### PR TITLE
Random line of code resets FanOpMode when UnitarySystem uses multi-speed heating coil and Setpoint control

### PR DESCRIPTION
**Pull request by rraustad**
Pull request overview
---------------------
 - Fixes an issue with set point controlled UnitarySystem where this line was resetting ContFanCycCoil to CycFanCycCoil
 - Not sure where this line came from or why it was added. Need to see what fails to know what else needs to change.

**NOTE: ENHANCEMENTS MUST FOLLOW A SUBMISSION PROCESS INCLUDING A FEATURE PROPOSAL AND DESIGN DOCUMENT PRIOR TO SUBMITTING CODE**

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [x] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [x] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [x] Perform a Code Review on GitHub
 - [x] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [x] CI status: all green or justified
 - [x] Check that performance is not impacted (CI Linux results include performance check)
 - [x] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

**Comment by rraustad** [comment]
This was a test case that turned into a hot fix. All good so merge at will.

**Comment by rraustad** [comment]
I'm trying to figure out what issue to post for the removal of this stray line of code. The line of code is in function simMultiSpeedCoils and only for heating coils. This function is only called for set point based control of UnitarySystems. There are no diffs, which is indicative of missing example files or unit tests. This line resets the FanOpMode to cycling fan so that would affect any Setpoint based control UnitarySystem using a "MultiSpeed" heating coil. That does narrow it down to a point where an issue can be described.

File: src/EnergyPlus/UnitarySystem.cc [hunk 1]
```diff
@@ -14395,7 +14395,6 @@ namespace UnitarySystems {
                 state.dataUnitarySystems->HeatingLoad = false;
             }
             LatLoad = 0.0;
-            this->m_FanOpMode = 1; // why is this here?
```
Comment (rraustad) [inline src/EnergyPlus/UnitarySystem.cc:14398]: The line of code that was removed as not necessary/not user input honorable.

File: src/EnergyPlus/VariableSpeedCoils.cc [hunk 2]
```diff
@@ -4698,7 +4698,7 @@ namespace VariableSpeedCoils {
                     if (varSpeedCoil.CondenserType == DataHeatBalance::RefrigCondenserType::Air) {
                         RatedSourceTempCool = OutTemp;
                     } else {
-                        RatedSourceTempCool = GetVSCoilRatedSourceTemp(state, DXCoilNum, ErrorsFound);
+                        RatedSourceTempCool = GetVSCoilRatedSourceTemp(state, DXCoilNum);
```
Comment (rraustad) [inline src/EnergyPlus/VariableSpeedCoils.cc:4701]: This is my Oops. The var is not used in this new function.

**Comment by rraustad** [comment]
@Myoldmopar fixing up a merge mistake and a random line of code that should not be there.

**Review by mjwitte — APPROVED** [review]
All good here. Merging.

**Pull request merged**